client/scripts/views/pages/profile/profile_header.ts

```diff
@@ -76,10 +77,20 @@ export interface IProfileHeaderState {
 }
 
 const ProfileHeader: m.Component<IProfileHeaderAttrs, IProfileHeaderState> = {
+  oninit: (vnode) => {
+    const { account } = vnode.attrs;
+    const step = m.route.param('step');
+    const joiningCommunity = m.route.param('joiningCommunity');
+    const joiningChain = m.route.param('joiningChain');
+    if (step) {
+      app.modals.removeAll();
+      app.modals.create({ modal: OnboardingModal, data: { joiningCommunity, joiningChain, address: account.address, step } });
+    }
+  },
   view: (vnode) => {
     const { account, refreshCallback, onOwnProfile, onLinkedProfile } = vnode.attrs;
     const showJoinCommunityButton = vnode.attrs.setIdentity && !onOwnProfile;
-    const isClaimable = !account || !account.profile || account.profile.isEmpty;
+    const isClaimable = !account || !account.profile;
```

Comment:
Are there cases where the profile will be Empty and also not owned by someone?

Reply:
Here is a Q/A when I asked to @dillchen about the `isClaimable`.
Q: If the address is already linked to an account but doesn't have any profile, should we show the claim address button?
A: No

So I need to fix based on this.
```
const isClaimable = !account
```
Is this okay?